### PR TITLE
D/mdot correction

### PR DESCRIPTION
**Pull request by mchurchf**
Documentation in the verification manual for the global mass-flow-rate correction for outflow BCs and for Robert's local moving-time-average Boussinesq reference temperature option with standard open BC for thermally stratified flow.  The test case is a simple slip channel with uniform 8 m/s inflow and a stratified temperature profile.  The flow should maintain this profile as it moves through the domain and exits.  The standard open BC with no other special options clearly distorts the flow.  The global mass-flow-rate correction option maintains the profiles and properly corrects mass flow rate imbalances.  The option for Boussinesq with the local moving-time-average reference temperatur9e also maintains the profiles.

### CODE REVIEW

File: docs/source/verification/verification.rst [hunk 1]
```diff
@@ -815,3 +815,82 @@ slight degradation in order-of-accuracy is noted for the thexahedral topology.
...
+
+Open Boundary Condition With Outflow Thermal Stratification
+----------------------------------------------------------------
+In situations with significant thermal stratification at the outflow of the domain, the standard open boundary
+condition alone is not adequate because it requires the specification of motion pressure at the boundary, and
+this is not known *a priori*.  Two solutions to this problem are: 1) to use the global mass flow rate correction
+option, or 2) to use the standard open boundary condition in which the buoyancy term uses a local time-averaged
+reference value, rather than a single reference value.
+
+We test these open boundary condition options on a simplified stratfied flow through a channel with slip walls.  The
```
Comment (spdomin) [inline docs/source/verification/verification.rst]: stratified

File: docs/source/verification/verification.rst [hunk 2]
```diff
@@ -815,3 +815,82 @@ slight degradation in order-of-accuracy is noted for the thexahedral topology.
...
+   Mass Balance Review:
+   Density accumulation: 0
+   Integrated inflow:    -188486
+   Integrated open:      188486
+   Total mass closure:   -9.82374e-06
+   A mass correction of: -1.22797e-08 occurred on: 800 boundary integration points:
+   Post-corrected integrated open: 188486
+
+The product of the point-wise mass correction of -1.22797e-06 over all 800 boundary integration points is -9.82374e-06,
+which agrees with the reported pre-correction mass-flow-rate error.
```
Comment (spdomin) [inline docs/source/verification/verification.rst]: We may need more precision in the integrated inflow/open output such that mass balance can be verified. The post-corrected could also increate precision for this.

Comment (mchurchf) [reply docs/source/verification/verification.rst]: Okay, I added precison

File: docs/source/verification/verification.rst (hunk 1 above)
Comment (mchurchf) [reply docs/source/verification/verification.rst]: Fixed.

**Comment by mchurchf** [comment]
Stefan, I made the changes you proposed above.  Thanks for your review.
